pkg/kubectl/resource_printer.go

```diff
@@ -337,20 +337,43 @@ type PrintOptions struct {
 // received from watches.
 type HumanReadablePrinter struct {
 	handlerMap map[reflect.Type]*handlerEntry
-	Options    PrintOptions
+	options    PrintOptions
 	lastType   reflect.Type
 }
 
 // NewHumanReadablePrinter creates a HumanReadablePrinter.
-func NewHumanReadablePrinter(options PrintOptions) *HumanReadablePrinter {
+func NewHumanReadablePrinter(pOptions PrintOptions) *HumanReadablePrinter {
 	printer := &HumanReadablePrinter{
 		handlerMap: make(map[reflect.Type]*handlerEntry),
-		Options:    options,
+		options:    pOptions,
 	}
 	printer.addDefaultHandlers()
 	return printer
 }
 
+// formatResourceName receives a resource kind, name, and boolean specifying
+// whether or not to update the current name to "kind/name"
+func formatResourceName(kind, name string, withKind bool) string {
+	if !withKind || kind == "" {
+		return name
+	}
+
+	return kind + "/" + name
+}
+
+// GetResourceName returns the currently set resource name
+func (h *HumanReadablePrinter) GetResourceName() string {
```

Comment:
Ah thanks for catching this. I will rename this method to `GetResourceKind`, as `name` and `kind` are not the same thing